src/libraries/System.ComponentModel.Annotations/tests/System.ComponentModel.Annotations.Tests.csproj

```diff
@@ -3,6 +3,7 @@
     <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
     <TargetFrameworks>$(NetCoreAppCurrent);net48</TargetFrameworks>
     <Nullable>disable</Nullable> <!-- Disable nullable attributes as some tests depend on them not being present. -->
+    <NoWarn>$(NoWarn);nullable</NoWarn>
```

Comment:
out of curiosity, what does this actually do when nullable annotations are disabled?

Reply:
Repeated question 😄 , [see the answer here](https://github.com/dotnet/runtime/pull/72568#discussion_r926832111)

Reply:
just noticed and was gonna resolve this 😆